README.md

```diff
@@ -51,3 +54,35 @@ This script will dispatch a remote job in dataflow with the specified configurat
 - `-r|--region`: the region where to execute the Dataflow job (for options see [Dataflow Locations](https://cloud.google.com/dataflow/docs/resources/locations))
 
 The job executed here will spawn a single Dataflow worker to consume the change stream.
+
+### BigQuery
+For BigQuery, you can execute the script like so:
+
+```bash
+./run_bigquery.sh \
+  --project <my-gcp-project> \
+  --instance <my-spanner-instance> \
+  --database <my-spanner-database> \
+  --metadata-instance <my-spanner-metadata-instance> \
+  --metadata-database <my-spanner-metadata-database> \
+  --change-stream-name <my-spanner-change-stream-name> \
+  --gcs-bucket <my-gcs-bucket> \
+  --big-query-dataset <my-big-query-dataset> \
+  --big-query-table-name <my-big-query-table-name> \
+  --region <my-dataflow-job-region>
+```
+
+This script will dispatch a remote job in dataflow with the specified configuration:
+
+- `-p|--project`: the Google Cloud Platform project id
+- `-i|--instance`: the Google Cloud Spanner instance id where the change stream resides
+- `-d|--database`: the Google Cloud Spanner database id where the change stream resides
+- `-mi|--metadata-instance`: the Google Cloud Spanner instance id where the Connector metadata tables will be created (we recommend it to be different than the change stream instance)
```

Comment:
Could you update this to remove the `(we recommend it to ...)`. That is not necessary for the instance, only for the metadata-database.